packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initializeFromChatEvent..st

```diff
@@ -3,4 +3,6 @@ initializeFromChatEvent: anEvent
 
 	super initializeFromChatEvent: anEvent.
 	self groupId: ((anEvent at: 'type') at: self groupKey).
-	self isChannel: ((anEvent at: 'type') at: 'is_channel').
+	((anEvent at: 'type') at: 'is_channel') ifNotNil: [
```

Comment:
I would prefer `at: ifAbsent:` here. maybe

`self isChannel: ((anEvent at: 'type') at: 'is_channel' ifAbsent: false)`

Reply:
removed this code as it was already fixed in a different commit